examples/A.hs

```diff
@@ -0,0 +1,7 @@
+{-# OPTIONS_GHC -fplugin Plugin.CheckImports #-}
```

Comment:
We should probably document somewhere that the package that exposes `Plugin.CheckImports` needs to be registered with GHC, for completeness? I kinda have a vague memory of having a lot of trouble finding clear, minimal directions on how to use plugins in the past.

Also, if I recall correctly, the name, `plugin` isn't significant, but the fact that the module `Plugin.CheckImports` exports only one symbol _is_ significant. Is that correct? Either way, we should write down the answer to these questions.

Reply:
Actually, that's something I have to check myself as well. Thanks for pointing out the need for such documentation. I will add that soon.

Reply:
added an explanation on how to use plugin.

Reply:
added explanation about plugin package